extension/partiql-extension-ion/src/lib.rs

```diff
@@ -114,45 +202,94 @@ mod tests {
         );
 
         // lob
-        assert_ion("{{ +AB/ }}", Value::Blob(Box::new(vec![248, 0, 127])));
+        assert_ion(
+            "{{ +AB/ }}",
+            ion_rs::element::Value::Blob(Bytes::from(vec![248, 0, 127])),
+            Value::Blob(Box::new(vec![248, 0, 127])),
+        );
         assert_ion(
             "{{ \"CLOB of text.\" }}",
+            ion_rs::element::Value::Clob(Bytes::from("CLOB of text.")),
             Value::Blob(Box::new("CLOB of text.".bytes().collect_vec())),
         );
 
         // list
-        assert_ion("[1,2,\"3\"]", partiql_list![1, 2, "3"]);
+        assert_ion(
+            "[1,2,\"3\"]",
+            ion_rs::element::Value::List(Sequence::new([
+                ion_rs::element::Value::Int(Int::I64(1)),
+                ion_rs::element::Value::Int(Int::I64(2)),
+                ion_rs::element::Value::String(Str::from("3")),
+            ])),
+            partiql_list![1, 2, "3"],
+        );
 
         // struct
-        assert_ion("{\"k\": []}", partiql_tuple![("k", partiql_list![])]);
+        assert_ion(
+            "{\"k\": [1,2,3]}",
+            ion_rs::element::Value::Struct(
+                Struct::builder()
+                    .with_field("k", ion_rs::element::List(Sequence::new([1, 2, 3])))
+                    .build(),
+            ),
+            partiql_tuple![("k", partiql_list![1, 2, 3])],
+        );
     }
 
     #[test]
     fn partiql_value_from_partiql_encoded_ion() {
-        assert_partiql_encoded_ion("null", Value::Null);
-        assert_partiql_encoded_ion("$missing::null", Value::Missing);
+        /*
+        assert_partiql_encoded_ion(
+            "null",
+            ion_rs::element::Value::Null(IonType::Null),
+            Value::Null,
+        );
+
+         */
```

Comment:
Do we need to keep this?

Reply:
Whoops. Yes, it shouldn't be commented out; fixing